src/client/pydaos/raw/daos_api.py

```diff
@@ -1514,13 +1514,12 @@ def create(self, poh, con_uuid=None, con_prop=None, cb_func=None):
             else:
                 ret = func(self.poh, self.uuid, ctypes.byref(self.cont_prop),
                            None)
-                if ret != 0:
-                    self.uuid = (ctypes.c_ubyte * 1)(0)
-                    raise DaosApiError(
-                        "Container create returned non-zero. RC: {0}".format(
-                            ret))
-                else:
-                    self.attached = 1
+            if ret != 0:
```

Comment:
this part of the patch is from another PR that has not yet landed, PR 1933. That one will land first and a merge with master will be done.